Dockerfile.ci

```diff
@@ -1,28 +1,11 @@
-ARG BASE_IMAGE_TAG=latest
-
-FROM codegram/decidim:$BASE_IMAGE_TAG
+FROM codegram/decidim:latest
 MAINTAINER deivid.rodriguez@riseup.net
 
-USER root
-
-RUN apt-get update \
-  && apt-get install -y unzip
-
-RUN DOCKERIZE_URL="https://circle-downloads.s3.amazonaws.com/circleci-images/cache/linux-amd64/dockerize-latest.tar.gz" \
-  && curl --silent --show-error --location --fail --retry 3 --output /tmp/dockerize-linux-amd64.tar.gz $DOCKERIZE_URL \
-  && tar -C /usr/local/bin -xzvf /tmp/dockerize-linux-amd64.tar.gz \
-  && rm /tmp/dockerize-linux-amd64.tar.gz \
-  && dockerize --version
-
-RUN CHROME_URL="https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb" \
```

Comment:
Why remove all these? We need them to run the tests 😕

Reply:
It's actually done via native `alpine` packages. Check the `apk add` lines that substitute them ->

Reply:
Ah cool!